docs/componentExtension.md

```diff
@@ -0,0 +1,70 @@
+## Customizing and Extending Components
+
+As mentioned in the Architecture.md when customizing or extending the sample app component, do not modify the packages within `@sfcc-bff` and `@sfcc-core`. These packages will be published and consumed via NPM. Instead, create a new custom package within the monorepo that registers itself with `@sfcc-core\core` and provides access to data from a third-party service.
```

Comment:
Should we have a link to `Architecture.md` file here?

Reply:
done